subworkflows/local/arg.nf

```diff
@@ -52,7 +52,7 @@ workflow ARG {
                 }
                 .set { ch_input_for_deeparg }
 
-        DEEPARG_PREDICT ( ch_input_for_deeparg, ch_deeparg_db.first() )
+        DEEPARG_PREDICT ( ch_input_for_deeparg, ch_deeparg_db )
```

Comment:
Why is deeparg modified in this PR?

Reply:
It was giving a big ugly warning that running `.first()` on that channel was useless, as it was already a value channel. Not sure why I hadn't seen it earlier...

Reply:
I forget now how `ch_deeparg_db` is created, is it always a value channel or does it depend on how it's created, i.e., is it only the case in the default scenario now?

Reply:
I moved the .first() to just the case when it was not a value channel (`fromPath`, the othercase it was output from module so allows re-use)